src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private static final int MAX_BOUND = 101;
+
+    public Ball getRandomBall() {
+        String color = String.valueOf(new ColorSupplier().getRandomColor());
+        int randomNumber = new Random().nextInt(MAX_BOUND);
```

Comment:
color supplier, random should be class-level private fields
don't use valueOf to get string representation of enum
![image](https://github.com/mate-academy/jv-lottery/assets/49767049/4bb6ab27-a55b-4849-947d-0838b3ccc61a)